samples/boards/nrf52/mesh/onoff-app/README.rst

```diff
@@ -0,0 +1,87 @@
+
+Bluetooth: Mesh OnOff Model
+###########################
+
+Overview
+********
+
+This is a simple application demonstrating a Bluetooth mesh multi-element node. Each element has a mesh onoff client and server
+model which controls one of the 4 sets of buttons and LEDs .
+
+Prior to provisioning, the board broadcasts an unprovisioned beacon containing a unique UUID. It is
+obtained from the device address set by Nordic in the FICR. Each button inverts the state of its
+corresponding LED and does not initiate any mesh activity.
+
+The models for button 1 and LED 1 are in the node's root element.
+The 3 remaining button/LED pairs are in elements 1 through 3.
+Assuming the provisioner assigns 0x100 to the root element,
+the secondary elements will appear at 0x101, 0x102 and 0x103.
+
+It's anticipated that after provisioning, the button clients would
+be configured to publish and the LED servers to subscribe.
+
+If a LED server is provided with a publish address, it will
+also publish its status on an onoff state change.
+
+Messages from a button to its corresponding LED are ignored as
+the LED's state has already been changed locally by the button client.
+
+
+Requirements
+************
+
+* This sample is specific to the Nordic nRF52840-PDK board.
+
+Building and Running
+********************
+
+This sample can be found under :file:`samples/boards/nrf52/mesh/onoff-app` in the
+Zephyr tree.
+
+The following commands build the application.
+
+.. code-block:: console
+
+   # cd samples/boards/nrf52/mesh/onoff-app
+   # mkdir build
+   # cd build
+   # cmake -DBOARD=nrf52840_pca10056 ..
+   # make
+   # make flash
+
+Provisioning is done using BlueZ' meshctl utility. Below is an example that binds
+button 2 and LED 1 to application key 1. It then configures button 2 to publish to group 0xc000
+and LED 1 to subscribe to the same group. Pressing button 2 will invert the state of both button 1 and
+2. Each button is hardwired to its corresponding LED. As one would expect with an actual switch,
+the switch toggle would unconditionally change the state of the its contacts.
+
+.. code-block:: console
+
+   discover-unprovisioned on
+   provision <discovered UUID>
+   menu config
+   target 0100
+   add-appkey 1
+   bind 0 1 1000              # bind appkey 1 to LED server on element 0 (unicast 0100)
+   sub-add 0100 c000 1000     # add subscription to group address c000 to the LED server
+   bind 1 1 1001              # bind appkey 1 to button 2 on element 1 (unicast 0101)
+   set-pub 0101 c000 1 0 1001 # publish button 2 to group address c000
+
+The meshctl utility maintains a persistent JSON database containing the mesh configuration. As additional nodes (boards) are provisioned, it
+assigns sequential unicast addresses based on the number of elements supported by the node. This example supports 4 elements per node.
+
+The first or root element of the node contains models for configuration, health and onoff. The secondary elements only
```

Comment:
add a comma after health:

    for configuration, health, and onoff.

Reply:
Fixed